enforcer-parent/commons/src/main/java/org/wso2/choreo/connect/enforcer/commons/model/ParameterResolver.java

```diff
@@ -63,7 +65,7 @@ public ParameterResolver(final String parameterTemplate) {
         final Matcher matcher = pattern.matcher(formattedURI);
         if (!matcher.matches()) {
             // Unlikely to occur as this pair is already matched within router.
-            logger.error("PathTemplate and RawPath is mismatched.");
+            logger.debug("PathTemplate:" + pathTemplate + " and RawPath:" + uriString + " is mismatched.");
```

Comment:
shall we use templated log format?
```suggestion
            logger.debug("PathTemplate: {}  and RawPath: {} is mismatched.", pathTemplate, uriString);
```